src/dmd/must_use.d

```diff
@@ -0,0 +1,198 @@
+/**
+ * Compile-time checks associated with the @mustUse attribute.
+ *
+ * Copyright: Copyright (C) 1999-2022 by The D Language Foundation, All Rights Reserved
+ * License:   $(LINK2 https://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:    $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/must_use.d, _must_use.d)
+ * Documentation:  https://dlang.org/phobos/dmd_must_use.html
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/must_use.d
+ */
+
+module dmd.must_use;
```

Comment:
Not thrilled about starting using _ in module and file names.

Reply:
Removed the _.